src/extension.js

```diff
@@ -31,10 +32,18 @@ Extension.getIcon = Utils.getIcon;
  */
 const ServiceIndicator = GObject.registerClass({
     GTypeName: 'GSConnectServiceIndicator',
-}, class ServiceIndicator extends PanelMenu.SystemIndicator {
+}, class ServiceIndicator extends QuickSettings.QuickMenuToggle {
 
     _init() {
-        super._init();
+        super._init({
+            label: 'GSConnect',
+            iconName: 'org.gnome.Shell.Extensions.GSConnect-symbolic',
+            toggleMode: true,
+        });
+
+        // Set QuickMenuToggle header.
+        this.menu.setHeader('org.gnome.Shell.Extensions.GSConnect-symbolic', 'GSConnect',
+            'Sync between your devices');
```

Comment:
```suggestion
            _('Sync between your devices'));
```

This string should be marked translatable.